src/Sarif.UnitTests/SarifCurrentToVersionOneTests.cs

```diff
@@ -1152,9 +1148,7 @@ public void SarifTransformerTests_ToVersionOne_ResultLocations()
                 ""region"": {
                   ""startLine"": 56,
                   ""startColumn"": 8,
-                  ""endColumn"": 78,
-                  ""offset"": 1124,
-                  ""length"": 22
+                  ""endColumn"": 78
```

Comment:
See line 1143 above. I can't comment that line in GitHub because it's not part of the change, but... part of Issue #93 is that `snippet` became a property of `region`.

Please do that in a separate PR -- this one is already complicated enough. But do make a note so it doesn't fall through the cracks!

Reply:
No WAIT! The move of `snippet` to `region` was part of oasis-tcs/sarif-spec/130, "Fix the location object". This is just another example, where we are silently deserializing an invalid v2 log file.

Please file the following issue: v2 log file with extra properties is silently deserialized.

Once you fix that, all the broken tests will pop to the surface, and you'll need to fix them as part of the same PR.

Reply:
1143 is v1 json
The issue still exists of course so I'll bug it.

Reply:
#933